src/calculator.py

```diff
@@ -168,9 +177,13 @@ def main():
             sess.commit()
             
         sess.close()
+        if killhandler.killed==True:
+            print ("Exited \"calculator\" gracefully")
```

Comment:
Please use logging.info instead of print, and change the text to "Termination signal received - exiting"